cupertino_gallery/lib/widgets/list_tile_page.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/cupertino.dart';
+
+class ListTilePage extends StatelessWidget {
+  const ListTilePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: const CupertinoNavigationBar(middle: Text('List Tile')),
+      child: Center(
+        child: ListView(
+          children: [
+            CupertinoListSection(
```

Comment:
`CupertinoListSection.insetGrouped` might be better to show. It became the default in SwiftUI since the CupertinoListSection class was created.